metrics/disk/cassandra_kubernetes/volume/local-storage.yaml

```diff
@@ -0,0 +1,6 @@
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: local-storage
```

Comment:
Can we change the name here to block-local-storage to be clear and similarly update at all other places. 
My idea is we could augment the test in the near future both to test block as well as virtio-fs.
For testing virtio-fs we could use another storage class setup using host volumes. What do you think?

Reply:
@amshinde changes applied